<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>866</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>981</width>
      <height>681</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>1</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="NextDay">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>133</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>NextDay</string>
      </property>
     </widget>
     <widget class="QPushButton" name="PreviousDay">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>133</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>PreviousDay</string>
      </property>
     </widget>
     <widget class="QPushButton" name="IsLeap">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>133</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>IsLeap</string>
      </property>
     </widget>
     <widget class="QPushButton" name="WeekNumber">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>133</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>WeekNumber</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>               Select any date</string>
      </property>
     </widget>
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>60</y>
        <width>201</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="DaysTillYourBirthday">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>60</y>
        <width>121</width>
        <height>171</height>
       </rect>
      </property>
      <property name="text">
       <string>DurationDay</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>250</y>
        <width>281</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>20</y>
        <width>431</width>
        <height>511</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>nextDay</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>previousDay</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>isLeap</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>WeekNumber</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="calculateDaysToBirthday">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>450</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Birthday</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_4">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>450</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string> 2</string>
     </attribute>
     <widget class="QPushButton" name="readfiledata">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>251</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string>Read data from file</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nextdayfile">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>250</y>
        <width>181</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>Next day file</string>
      </property>
     </widget>
     <widget class="QPushButton" name="differenceDay">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>270</y>
        <width>181</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Day difference</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>20</y>
        <width>451</width>
        <height>611</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Dates file</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>NextDay file</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>  Between dates</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>  Day difference  </string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="adddays">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>370</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>addData</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>440</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="removeDate">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>380</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>removeDate</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_3">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>380</y>
        <width>61</width>
        <height>50</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1028</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
